paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/PlaygroundTestDriver.kt

```diff
@@ -1145,7 +1145,14 @@ internal class PlaygroundTestDriver(
                                 .text(testParameters.isSetupMode),
                             className = "android.widget.TextView",
                             device = device
-                        ) {}.click()
+                        ) {
+                            override fun click() {
+                                if (testParameters.paymentMethodCode == "wechat_pay") {
+                                    wait(5000)
```

Comment:
can you use a compose.waitUntil here instead of a hard-coded delay?

Reply:
I think if you do this, then maybe you could just unconditionally add this, instead of having to add it just for wechat pay. And then we won't see issues like this for other LPMs in the future (if it started happening for other LPMs)

Reply:
No because the web view isn't compose based

Reply:
Yeah that makes sense. Do we want to add a potentially unnecessary delay for all e2e test though?